packages/random/random.pony

```diff
@@ -4,14 +4,20 @@
 The Random package provides support generating random numbers. The package
 provides random number generators you can use in your code, a dice roller and
 a trait for implementing your own random number generator.
+
+If your application does not require a specific generator, use Rand.
```

Comment:
I think with the addition of the type alias, that a usage example should be included for:

1. "how you use Rand"
2. "how to switch to a specific generator"

Random numbers are a very common thing for beginners to use and the directions here are not particularly begineer friendly.